staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/BUILD

```diff
@@ -1,10 +1,4 @@
-package(default_visibility = ["//visibility:public"])
-
-load(
-    "@io_bazel_rules_go//go:def.bzl",
-    "go_library",
-    "go_test",
-)
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
```

Comment:
this change looks weird... did the generator do this?

Reply:
will check. It did some weird other things, like not updating the file.

Reply:
fixed.